### PR TITLE
Updated accessible_output2 hook

### PR DESCRIPTION
**Pull request by NicklasTegner**

### CODE REVIEW

File: news/45.update.rst
Comment (bwoodsend) [inline news/45.update.rst:1]: Update ``accessible_output2`` hook.

**Review by bwoodsend — CHANGES_REQUESTED** [review]

**Review by NicklasTegner — COMMENTED** [review]
Should be done

**Comment by bwoodsend** [comment]
Umm, did you push the wrong branch? There are no commits since my last comment and the news fragment is still unchanged.

**Review by bwoodsend — APPROVED** [review]
Yay, again I'll wait for @Legorooj's approval before merging.

**Comment by NicklasTegner** [comment]
No problem. Thanks for being so patient with me :)

**Comment by Legorooj** [comment]
What's the rationale for this change? Why use `datas` and not `binaries`?

**Comment by bwoodsend** [comment]
The package contains binaries for all platforms and it looks like our binary scanner doesn't handle DLLs for the wrong OS. On Windows you get huge numbers of warnings. On macOS I'm told the DLLs get excluded.

**Comment by NicklasTegner** [comment]
My rationale for this change was, that I made it directly after the changes to #44  because, when using "binaries" in that original hook, the build would fail on mac (due to, there being dlls in the librarys directory) with an invalid/unknown header (don't remember the error)
@bwoodsend  this library only contains dlls, are you maybe thinking of "sound_lib" #44

**Comment by NicklasTegner** [comment]
Here is the traceback I'm getting on MacOS.
Notice, that when PyInstaller encounter a non dylib file when searching for binaries, the following thing happens (just like in #44  )
So just like in #44  switching them to "data" instead of "binaries" is a workaround, that fixes both instances.

Note: I have removed some of the PyInstaller output that was working fine.

> 4702 INFO: running Analysis Analysis-00.toc
4707 INFO: Analyzing /Users/nicklas/Desktop/python-projects/ao2-test/test.py
4746 INFO: Processing pre-safe import module hook win32com from '/usr/local/lib/python3.8/site-packages/_pyinstaller_hooks_contrib/hooks/pre_safe_import_module/hook-win32com.py'.
4932 INFO: Processing module hooks...
4932 INFO: Loading module hook 'hook-accessible_output2.py' from '/usr/local/lib/python3.8/site-packages/_pyinstaller_hooks_contrib/hooks/stdhooks'...
5166 INFO: Looking for dynamic libraries
Traceback (most recent call last):
  File "/usr/local/bin/pyinstaller", line 8, in <module>
    sys.exit(run())
  File "/usr/local/lib/python3.8/site-packages/PyInstaller/__main__.py", line 114, in run
    run_build(pyi_config, spec_file, **vars(args))
  File "/usr/local/lib/python3.8/site-packages/PyInstaller/__main__.py", line 65, in run_build
    PyInstaller.building.build_main.main(pyi_config, spec_file, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/PyInstaller/building/build_main.py", line 720, in main
    build(specfile, kw.get('distpath'), kw.get('workpath'), kw.get('clean_build'))
  File "/usr/local/lib/python3.8/site-packages/PyInstaller/building/build_main.py", line 667, in build
    exec(code, spec_namespace)
  File "/Users/nicklas/Desktop/python-projects/ao2-test/test.spec", line 6, in <module>
    a = Analysis(['test.py'],
  File "/usr/local/lib/python3.8/site-packages/PyInstaller/building/build_main.py", line 242, in __init__
    self.__postinit__()
  File "/usr/local/lib/python3.8/site-packages/PyInstaller/building/datastruct.py", line 160, in __postinit__
    self.assemble()
  File "/usr/local/lib/python3.8/site-packages/PyInstaller/building/build_main.py", line 466, in assemble
    self.binaries.extend(bindepend.Dependencies(self.binaries,
  File "/usr/local/lib/python3.8/site-packages/PyInstaller/depend/bindepend.py", line 230, in Dependencies
    for lib, npth in selectImports(pth, xtrapath):
  File "/usr/local/lib/python3.8/site-packages/PyInstaller/depend/bindepend.py", line 504, in selectImports
    dlls = getImports(pth)
  File "/usr/local/lib/python3.8/site-packages/PyInstaller/depend/bindepend.py", line 745, in getImports
    return _getImports_macholib(pth)
  File "/usr/local/lib/python3.8/site-packages/PyInstaller/depend/bindepend.py", line 628, in _getImports_macholib
    m = MachO(pth)
  File "/usr/local/lib/python3.8/site-packages/macholib/MachO.py", line 118, in __init__
    self.load(fp)
  File "/usr/local/lib/python3.8/site-packages/macholib/MachO.py", line 133, in load
    self.load_header(fh, 0, size)
  File "/usr/local/lib/python3.8/site-packages/macholib/MachO.py", line 167, in load_header
    raise ValueError("Unknown Mach-O header: 0x%08x in %r" % (header, fh))
ValueError: Unknown Mach-O header: 0x4d5a9000 in <_io.BufferedReader name='/usr/local/lib/python3.8/site-packages/accessible_output2/lib/nvdaControllerClient64.dll'>

**Comment by Legorooj** [comment]
@NicklasTegner can you please rework this PR to use the `collect_data_files` function's `includes` and `excludes` arguments? And assign it to `binaries` as well.

**Pull request closed without merging**
